src/csvcubed/readers/skoscodelistreader.py

```diff
@@ -30,8 +31,10 @@ def extract_code_list_concept_scheme_info(
       `concept_uri_template` uses the standard `notation` uri template variable even if the underlying file uses a
        different column name.
     """
+
     csvw_rdf_manager = CsvwRdfManager(code_list_csvw_path)
-    result = select_table_schema_properties(csvw_rdf_manager.rdf_graph)
+    csvw_inspector = CsvWInspector(csvw_rdf_manager.rdf_graph, code_list_csvw_path)
```

Comment:
Instead of creating it yourselves, can you access it via `csvw_rdf_manager.csvw_inspector` so we (hopefully) only ever create one `CsvWInspector` per CSV-W loaded? 

If we have multiple instances of the `CsvWInspector` class, we'll still end up running queries multiple times even with the `@cached_property`.